terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionLevelResolverFilter.java

```diff
@@ -32,6 +32,9 @@ protected void doFilterInternal(HttpServletRequest request,
             HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
 
         String path = request.getServletPath();
+        if (path == null || path.length() == 0) {
+            path = request.getPathInfo();
+        }

```

Comment:
@ikeyat 
This is related this PR.
In Websphere(Liberty Profile), `HttpServletRequest#getServletPath()` return empty string unlike other servers, `HttpServletRequest#getPathInfo()` return the path to be expected.